README.md

```diff
@@ -31,14 +31,16 @@ Use [@BotFather](https://t.me/botfather) to create a Telegram bot and get your b
 
 To get your Perspective API token, follow the [getting started guide](https://developers.perspectiveapi.com/s/docs-get-started), and then follow the "Enable the API" section.
 
-## Start w/Docker
-To simplify getting started with this bot, a `docker-compose.yml` file has been included. Ensure you have installed [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/install/). Once installed, run `docker-compose up -d` to start the environment. This will do a few things:
+## Start with Docker
+To simplify getting started with this bot, a `docker-compose.yml` file has been included. Ensure you have installed [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/install/). Once installed, run `yarn start:docker` to start the environment. The containers will do a few things:
```

Comment:
Not everyone has yarn installed or even NPM. The whole reason for incorporating Docker in the first place was to prevent cluttering local systems.

Reply:
Is this actually a situation that occurs? Don't users have to have local dependencies installed in order to get any sort of linting or formatting support?

Reply:
Although you and I use Yarn, not everyone does. I don't think there's any harm in including the command to start the docker environment but I do think the base command should be included in the README

Reply:
```suggestion
To simplify getting started with this bot, a `docker-compose.yml` file has been included. Ensure you have installed [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/install/). Once installed, run `docker-compose up -d` to start the environment. If you have `npm` and `yarn` installed, you can use `yarn start:docker`. The containers will do a few things:
```

Reply:
You're right, we should include the raw command. Although I'm still curious how development works without node/npm/yarn locally. Is that still possible?

Reply:
Or more specifically, is it possible to still apply prettier formatting or run lint fixes?

Reply:
I don't think eslint or prettier work without node, and I feel pretty strongly that we should be applying both to any code before it's merged. 

I think we should add a section here (and eventually create a while CONTRIBUTING file) requiring contributors to run their code through prettier and eslint using out project's settings before opening a PR.

Reply:
I'm sure it's possible to get an editor to talk to the Node container so ESLint and Prettier are only being used there, but that seems like overkill. For now, having both commands listed seems good enough.